NewForce-Cohort-4/react-nutshell-house-baboonpaw
# Dtb modify chat messages

# Description
Added an edit feature to the messaging component. 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- pull down new branch
- login
- click on messaging nav link
- add a new message. Make sure that the message form clears once the "+" button is clicked and that the message appears at the bottom of the container. 
- click on the "..." button next to your newly created message. Make sure that the message is replaced with a form which has the original message text in the input field. 
- edit the text. Click "+".
- You should see the edited message in place of the original.
# Checklist:
